commit 5e4ea9d6f5896af89553ca058f95e3e6bc3111b1 (refs/changes/21/2921/4)
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2017-11-09T16:07:43+01:00 (1 year, 11 months ago)
    
    SAPI-295 SAPI crashes if MASTER_MORAY_IP is a hostname
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/lib/server/stor/moray.js b/lib/server/stor/moray.js
index cb26743..69aa31b 100644
--- a/lib/server/stor/moray.js
+++ b/lib/server/stor/moray.js
@@ -111,13 +111,11 @@ MorayStorage.prototype.init = function init(cb) {
                 port: config.master_port
             });
         } else {
-           this.master = moray.createClient({
-                connectTimeout: config.connectTimeout,
+            assert.equal(2020, config.master_port, 'config.master_port');
+            this.master = moray.createClient({
                 log: log,
                 srvDomain: config.master_host,
-                cueballOptions: {
-                    defaultPort: config.master_port
-                }
+                connectTimeout: config.connectTimeout
             });
         }
 
